src/components/BackButton.spec.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+import BackButton from './BackButton'
+import userEvent from '@testing-library/user-event'
+
+describe('BackButton', () => {
+  it('contains a text', () => {
+    const onNavigate = jest.fn()
+    render(<BackButton onClick={onNavigate}>Back</BackButton>)
+
+    const button = screen.getByRole('button')
+    expect(button).toBeInTheDocument()
+  })
+
+  it('calls onClick', () => {
+    const onNavigate = jest.fn()
+    render(<BackButton onClick={onNavigate}>Back</BackButton>)
+
+    const button = screen.getByRole('button')
+    userEvent.click(button)
+
+    expect(onNavigate).toHaveBeenCalledTimes(1)
```

Comment:
Happy to see that this test works now. Could you please share, what was the reason for this not working in the first version?

Reply:
The correct Proptype was "onClick" and I have supplemented jest.fn() before the first render.